rfc-029-source-code-should-be-open-source-by-default.md

```diff
@@ -0,0 +1,144 @@
+
+## A note on naming
+
+In this RFC we use the term “open source”, to be consistent with the
+language which the UK government uses to talk about software. We take
+our definition of open source software as being to equal that of [free
+software](https://en.wikipedia.org/wiki/Free_software)—software which
+allows users to run, study, redistribute or improve it.
+
+## Problem
+
+dxw creates a lot of source code repositories—for tools that we use
+internally, and for client work. When writing new code, we have a
+choice between making it available to the
+public and keeping it private for our own use or that of the client.
+
+Sometimes the code that we write might be useful to the rest of the
+world—it solves a common problem, or it provides inspiration for how
+to solve a problem. We don’t usually work on projects that require
+secrecy. We already make most of our repositories public—most notably
+the Playbook—but until now we haven’t had an official policy, although
+we’ve [tried to](https://github.com/dxw/playbook/issues/12) in the
+past. Most of our client projects are also in public repositories.
+
+We’ve [already talked
+about](rfc-018-our-decision-making-process-is-public.md) why we make
+our RFC decision making process open. Those reasons apply here, too.
+
+Allowing the team to work in the open also has benefits for individual
+developers. They:
+
+- can show their work to friends and family—show the thing instead of
+  talking about the thing
+- know that they’re investing their energy in something maximally
+  useful and long-lasting
+- can build up a public portfolio of work
+- can talk about their work without worrying about restrictions like
+  NDAs
+
+We should also make a simultaneous decision about software
+licensing. Whilst it’s theoretically possible to make a GitHub
+repository public without an explicit license, we shouldn’t do this,
+because:
+
+- having a public repository without a licence means that legally
+  [nobody can do anything useful with it](https://choosealicense.com/no-permission/),
+  so there’s not much point in doing so
+- releasing software without explicit no-liability, no-warranty
+  clauses might cause legal problems for us one day
+
+We had a brief discussion about this as a team and the MIT Licence is
+the one that came up the most. It’s a simple licence which allows
+people to do what they wish with the code and removes any potential
+legal issues for us. The other licence that came up is some version of
+the [GPL](https://choosealicense.com/licenses/gpl-3.0/), but even if
+it were something we wished to use, we wouldn’t have the resources to
+monitor for and pursue license violations.
+
+## Proposal
+
+All new GitHub respositories in the dxw organisation SHOULD be public,
+unless they directly relate to a client project. When these
+repositories are public, they SHOULD contain a copy of the [MIT
+License](https://choosealicense.com/licenses/mit/) in a `LICENSE` file
+in the repository’s root.
```

Comment:
Who owns the copyright on the licence? `(c) dxw` or do we need a different value?